_health-care/_js/components/financial-assessment/Child.jsx

```diff
@@ -29,7 +29,7 @@ class Child extends React.Component {
 
         <div className="row">
           <div className="small-12 columns">
-            <ErrorableSelect required
+            <ErrorableRadioButtonGroup required
```

Comment:
I don't think this is the right question we want to make into a radio button. It's the checkbox question "Do you have any children to report?" in `ChildInformationSection.jsx`.